src/shapes/Group.ts

```diff
@@ -92,7 +92,12 @@ export class Group
    * Used to allow targeting of object inside groups.
    * set to true if you want to select an object inside a group.\
    * **REQUIRES** `subTargetCheck` set to true
+   * This will be not removed but slowly replaced with a method setInteractive
+   * that will take care of enabling subTargetCheck and necessary object events.
+   * There is too much attached to group interactivity to just be evaluated by a
```

Comment:
I disagree
IMO there is no need
In some open PR I made `interactive` include logically `subTargetCheck` because that is how we use it so the code should reflect it.
It makes it easier to deal with and restores `subTargetCheck` to what it originally was.
So the dev can flag `interactive` and that is enough

Reply:
There is no need to register events for objects that will never move or be target of actions.
Group interactivity was put in as a part of the layout manager while it has nothing to do with layouting, nor layouting should take care of attaching event for detecting movements.
Those 2 parts now are strongly bound together and i can't think of my layout logic without having to think of the interaction and registration part.

Registering everything all the time doesn't bring performance issues, but is also completely unnecessary.